packages/block-library/src/cover/save.js

```diff
@@ -144,6 +147,17 @@ export default function save( { attributes } ) {
 					data-object-position={ objectPosition }
 				/>
 			) }
+			{ isEmbedVideoBackground && url && (
+				<figure
+					className={ clsx(
+						'wp-block-cover__video-background',
```

Comment:
Is this classname needed or should we clearly separate the "embed" from the "video" background rendering/classes?